eos/effects/usemissiles.py

```diff
@@ -4,7 +4,27 @@
 # Modules from group: Missile Launcher Heavy (12 of 12)
 # Modules from group: Missile Launcher Rocket (15 of 15)
 # Modules named like: Launcher (151 of 151)
-type = 'active'
-def handler(fit, module, context):
+type = 'active', "projected"
+
+
+def handler(fit, src, context):
     # Set reload time to 10 seconds
-    module.reloadTime = 10000
+    src.reloadTime = 10000
+
+    if "projected" in context:
+        if src.item.group.name == unicode("Missile Launcher Bomb"):
```

Comment:
>It's stored as unicode, yes, but it isn't unicode. Everything in that string could be ASCII.

`if src.item.group.name == "Missile Launcher Bomb":`

Doesn't match.

I'll change it to `u'` format.